packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -1231,6 +1232,10 @@ class CliServiceProvider extends ServiceProviderCore implements ServiceProvider
     }
     throw new MongoshInternalError('Cannot call watch with defined collection but undefined db');
   }
+
+  get driverMetadata(): ClientMetadata | undefined {
+    return this.mongoClient.topology?.clientMetadata;
```

Comment:
Do we need to use this, like is it a public API? Although probably `require('mongodb/package.json').version` wouldn't work for bundlers

Reply:
From their typescript definitions:

```js
/**
 * A container of server instances representing a connection to a MongoDB topology.
 * @public
 */
export declare class Topology extends EventEmitter {
```

The `@public` sounds good to me. (They also have `TopologyPrivate`, which I guess are the bits we’re not supposed to touch.)